lib/services/appmanager-service.ts

```diff
@@ -48,10 +55,49 @@ class AppManagerService implements IAppManagerService {
 
 			this.$logger.info("Successfully uploaded package.");
 
-			var tamUrl = util.format("%s://%s/appbuilder/Services/tam", this.$config.AB_SERVER_PROTO, this.$config.AB_SERVER);
-			this.$logger.info("Go to %s to manage your apps.", tamUrl);
-			this.$opener.open(tamUrl);
+			this.openAppManagerStore();
+		}).future<void>()();
+	}
+
+	public openAppManagerStore(): void {
+		var tamUrl = util.format("%s://%s/appbuilder/Services/tam", this.$config.AB_SERVER_PROTO, this.$config.AB_SERVER);
+		this.$logger.info("Go to %s to manage your apps.", tamUrl);
+		this.$opener.open(tamUrl);
+	}
+
+	private configureLivePatchPlugin(): IFuture<void> {
+		return (() => {
+			var plugins = this.$pluginsService.getInstalledPlugins();
+			if(!_.any(plugins, plugin => plugin.data.Identifier === AppManagerService.LIVEPATCH_PLUGIN_ID)) {
+				this.$logger.warn("AppManager LiveSync plugin is not enabled in your project. Enabling it now for release configuration.");
+				this.$pluginsService.addPlugin(AppManagerService.LIVEPATCH_PLUGIN_ID).wait();
+				this.$logger.info("AppManager LiveSync plugin is now enabled for release configuration.");
+			}
+		}).future<void>()();
+	}
+
+	public publishLivePatch(platforms: string[]): IFuture<void> {
+		return (() => {
+			platforms = _.map(platforms, platform => this.$mobileHelper.normalizePlatformName(platform));
+			var cachedOptionsRelease = commonOptions.release;
+			commonOptions.release = true;
+			this.configureLivePatchPlugin().wait();
+			var message = "If you have not published a LivePatch-enabled major version of this app before, you will not be able to distribute a LivePatch for it." + os.EOL +
+				"To create a new major version enabled for LivePatch, run the `$ appbuilder appmanager upload <Platform>` command";
+			this.$logger.warn(message);
```

Comment:
We can make the warn to accept an array of strings (along with string with union types) and make the concatenation there. This is for another PR though.